x-pack/plugins/cases/public/components/case_view/case_view_tabs.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiBetaBadge, EuiSpacer, EuiTab, EuiTabs } from '@elastic/eui';
+import React, { useCallback, useMemo } from 'react';
+import styled from 'styled-components';
+import { CASE_VIEW_PAGE_TABS } from '../../../common/types';
+import { useCaseViewNavigation } from '../../common/navigation';
+import { useCasesContext } from '../cases_context/use_cases_context';
+import { EXPERIMENTAL_DESC, EXPERIMENTAL_LABEL } from '../header_page/translations';
+import { useCasesTitleBreadcrumbs } from '../use_breadcrumbs';
+import { UseDescriptionHandler } from '../user_actions/use_description_handler';
+import { ACTIVITY_TAB, ALERTS_TAB } from './translations';
+import type { Case } from '../../../common';
+import { useOnUpdateField } from './use_on_update_field';
+
+const ExperimentalBadge = styled(EuiBetaBadge)`
+  margin-left: 5px;
+`;
+
+export interface CaseViewTabsProps {
+  caseData: Case;
+  activeTab: CASE_VIEW_PAGE_TABS;
+}
+
+export const CaseViewTabs = React.memo<CaseViewTabsProps>(({ caseData, activeTab }) => {
+  const { features } = useCasesContext();
+  const { navigateToCaseView } = useCaseViewNavigation();
+  const { onUpdateField, isLoading, loadingKey } = useOnUpdateField({
+    caseData,
+  });
+
+  useCasesTitleBreadcrumbs(caseData.title);
```

Comment:
This is unrelated to the tabs and it is better if it is part of the `CaseViewPage` component. If in the future we change or remove the tabs we will lose the ability to update the breadcrumbs.